README_ES.md

```diff
@@ -0,0 +1,199 @@
+[![TShock](https://img.shields.io/badge/TShock5.2.0-2B579A.svg?&logo=TShock&logoColor=white)](https://github.com/Pryaxis/TShock)
+
+**[简体中文](README.md)** | **[English](README_EN.md)** | **&gt; Spanish/Español &lt;**
+
+</div>
+
+## Introduccion
+- Este es un repositorio dedicado a recopilar e integrar complementos "TShock".
+- Algunos de los complementos de la biblioteca se recopilan de Internet y se descompilan.
+- Debido a la naturaleza especial del proyecto, puede causar infracción. Si hay alguna infracción, por favor contáctenos para resolverla.
+- Continuaremos recopilando complementos de "TShock" de alta calidad, actualizándolos de manera oportuna y estando al día con la última versión de "TShock".
+- Si desea unirse a nosotros, siga las "Notas del desarrollador" y envíe una "Solicitud de extracción" a este repositorio.
+
+
+## Notas de usuario
+
+- Tenga en cuenta que algunos complementos pueden requerir dependencias; consulte la lista a continuación para instalar las dependencias.
+- Cada complemento tiene una nota de uso; haga clic en el hipervínculo en la lista a continuación para ver las instrucciones específicas.
+- Se dice que a las personas a las que les gusta destacar repositorios, sus complementos no son fáciles de generar errores.
+
+## Descargas
+
+- Github Release: [Plugins.zip](https://github.com/UnrealMultiple/TShockPlugin/releases/download/V1.0.0.0/Plugins.zip)
+- Gitee Release: [Plugins.zip](https://gitee.com/kksjsj/TShockPlugin/releases/download/V1.0.0.0/Plugins.zip)
+
+#### AutoPluginManager
+    /apm l Lista todos los plugins
+    /apm i <plugin name> Una vez - Instalar Plugin
+    /apm u [plugin name] Ver si el plugin tiene actualizacion
+
+## Notas de Desarrollador
+
+> Estandarización de codigo
+
+- No utilice nombres de variables chinos.
+- No utilice funciones peligrosas.
+- Evite el uso de subprocesos múltiples siempre que sea posible.
+- No dejar puertas traseras en los complementos.
+- Incluya un archivo README.md con cada proyecto de complemento.
+
+## Comentarios
+
+> Cualquier comentario, sugerencia o mejora en esta biblioteca de códigos se considerará contribución pública y podrá incluirse en esta biblioteca de códigos a menos que se indique explícitamente lo contrario.
+
+- Si hay un error, proporcione la información relevante del sistema, la versión de TShock y el proceso de reproducción del error en la página "problema" de GitHub.
+
+### Plugins Recolectados
+
+> Haga clic en los hipervínculos para ver la descripción detallada del complemento
+
+> [!NOTE]
+> Es posible que la documentación del complemento en inglés no se actualice tan rápidamente como la documentación del complemento en chino.
+> Intente consultar la documentación china siempre que sea posible.
+
+<Details>
+<Summary>Listado de Plugins</Summary>
+
+
+md
+Copiar código
```

Comment:
`Copiar código`?

Reply:
It's because I was copying from the markdown website and by mistake I copied that